### PR TITLE
Bump dependencies versions

### PR DESCRIPTION
**Pull request by fflorent**

### CODE REVIEW

File: package.json
Comment (berhalak) [inline package.json:96]: Can you use explicit versions numbers (here without `^`). Sometimes it is hard to track down the bug due to mismatch in local and prod version of some dependency.

Comment (fflorent) [reply package.json]: Done!

File: app/client/ui/HomeImports.ts
Comment (fflorent) [inline app/client/ui/HomeImports.ts:44]: FWIW, I achieved to hit a breakpoint here. The `workspaceId` equals to `"unsaved"` when the user is not logged in but is allowed to upload documents (temporary).

The values are the one that are expected.

File: sandbox/watch.sh
Comment (jordigh) [inline sandbox/watch.sh:21]: This is a pretty cool way to replace the abandoned `catw` package.

**Review by jordigh — APPROVED** [review]

**Comment by fflorent** [comment]
@jordigh Thanks for your review. Do you wait for another review before merging?

Otherwise I do not have the rights to merge myself.

**Comment by jordigh** [comment]
@fflorent I need to first verify deploying to our own Grist SAAS before I merge.

**Comment by fflorent** [comment]
@jordigh Sounds good! :+1:

**Comment by jordigh** [comment]
Thanks for your patience. After some internal testing, we are satisfied with the state of this PR.

**Pull request merged**
